Sprint-2/2-mandatory-debug/1.js

```diff
@@ -1,13 +1,19 @@
 // Predict and explain first...
 //  =============> write your prediction here
+// there is an error on line 5 and 6  
+//function sum(a, b) {
+// //return;
+//  a + b;
+// }
 
-function sum(a, b) {
-  return;
-  a + b;
-}
-
-console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+// console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
 
 // =============> write your explanation here
+//the return statement is not returning any value. a+b declared after return.
```

Comment:
You are right about the return but be careful with second part, `a + b` is not not a declaration, it's an expression.